### PR TITLE
Enable article level

### PR DESCRIPTION
**Pull request by davidskalinder**
Okay, I've now split the article-level functionality into two hopefully much cleaner PRs.  This one should only enable the functionality to sit in the background with no visibility for users.

It does require a database migration however, namely, the new `coder_article_annotation` table; running `init_db()` should create this successfully (even, I believe, from a live database, although I haven't confirmed this recently).

Other than that, I've copied the `event-creator.html` and `event-creator-block.html` templates from the campus protests project, and uncommented the lines of the controller that display the (event-level) article description in the center panel for events without an event description.

So when I test this on a migrated database, everything looks and acts like it did back in the pre-COVID parts of '19 as far as I can see.  But of course the article-level functionality is waiting under the hood.

Tl;dr: don't accept this PR until database migration is sorted, and after that make sure the PR is tested on a non-critical deployment first; but it should enable all the back-end stuff to code at article-level without actually activating any of it.

### CODE REVIEW

**Comment by davidskalinder** [comment]
So I can't figure out how or why I closed this, and I'm assuming it was an error.  (Maybe I deleted the branch and re-created it without noticing that it automatically closed the PR?)  Anyway, I'm going to see if I can reopen it without ruining anything...

**Comment by davidskalinder** [comment]
Ugh, yes, GitHub tells me `The enable_article_level branch was force-pushed or recreated`.  Rather than do [all this](https://gist.github.com/robertpainsi/2c42c15f1ce6dab03a0675348edd4e2c), I'm just going to open a new PR from the recreated branch.
